src/IO/Exporter/Python/__init__.py

```diff
@@ -1,4 +1,49 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
+from typing import Sequence, Type
+
 from ._Pybindings import *
+
+
+def interpolate_tensors_to_points(
```

Comment:
Why not put this next to `interpolate_to_points` so it's exposed?

Reply:
This is next to `interpolate_to_points`, which is a binding brought in through the `from ._Pybindings import *` in this file. Both can be imported as `from spectre.IO.Exporter import interpolate_[tensors_]_to_points`.